src/plugin.js

```diff
@@ -1,14 +1,14 @@
+import { classes } from '@warp-ds/component-classes/classes';
```

Comment:
I'm a bit confused why these classes are used by default under `externalClasses`. Are they going to be replaced with an Eik url like we do with resets.min.css?